ivy_tests/test_ivy/test_functional/test_core/test_device.py

```diff
@@ -585,11 +585,21 @@ def test_profiler(device, call):
         time.sleep(1)  # required by MXNet for some reason
 
 
+@given(num=st.integers(0, 5))
+def test_num_arrays_on_dev(num, device):
+    arrays = {"x": []}
+    for _ in range(num):
```

Comment:
I'm being picky now, but you can actually do this in one line: `arrays = {"x": [ivy.array(np.random.uniform(size=2)) for _ in range(num)]}` :)